### DISCUSSION TITLE
构建镜像时提示“容器运行时不支持此功能”

### DISCUSSION THREAD

**Discussion by llm-infer** [kubesphere/kubesphere#6037] [Q&A]
构建镜像时提示“容器运行时不支持此功能”，是因为我的容器运行时是**containerd**而不是**docker**吗？

环境如下：
kubesphere版本：3.4.0
容器运行时：containerd.io 1.6.28
kubernetes版本：1.27.1

**Comment by weiliang-ms** [comment]
s2i好像只支持docker
